cloud/services/compute/subnets/reconcile.go

```diff
@@ -68,6 +72,11 @@ func (s *Service) createOrGetSubnets(ctx context.Context) ([]*compute.Subnetwork
 				return subnets, err
 			}
 
+			if s.scope.IsSharedVpc() {
+				logger.Error(err, "Shared VPC is enabled. Error looking for subnetwork", "name", subnetSpec.Name)
```

Comment:
same comment as above, the error should make it clear that existing subnets are required but weren't found